server/netty-server/src/main/scala/sttp/tapir/server/netty/internal/NettyToResponseBody.scala

```diff
@@ -40,7 +46,32 @@ class NettyToResponseBody extends ToResponseBody[NettyResponse, NoStreams] {
         val fileRange = v.asInstanceOf[FileRange]
         (ctx: ChannelHandlerContext) => ChunkedFileNettyResponseContent(ctx.newPromise(), wrap(fileRange))
 
-      case _: RawBodyType.MultipartBody => throw new UnsupportedOperationException
+      case m: RawBodyType.MultipartBody =>
+        val buffers: List[ByteBuf] = v
+          .asInstanceOf[List[RawPart]]
+          .flatMap(part =>
+            m.partType(part.name)
+              .map(bodyType => convertToBuffs(bodyType, part))
+          )
+        (ctx: ChannelHandlerContext) => ByteBufNettyResponseContent(ctx.newPromise(), Unpooled.wrappedBuffer(buffers: _*))
+    }
+  }
+
+  private def convertToBuffs(bodyType: RawBodyType[_], part: Part[Any]): ByteBuf = {
```

Comment:
Consider placing `convertToBuffs` after `wrap`s, so that this file is easier to read.

Reply:
done